sound/hda/intel-sdw-acpi.c

```diff
@@ -57,16 +57,17 @@ static int
 sdw_intel_scan_controller(struct sdw_intel_acpi_info *info)
 {
 	struct acpi_device *adev = acpi_fetch_acpi_dev(info->handle);
-	u8 count, i;
+	u32 count;
+	int i;
 	int ret;
```

Comment:
nitpick: Would it be better to move `int i;` after `int ret;`?
Like:
```
u32 count;
int ret;
int i;
```

Reply:
done